lib/src/scaffold.dart

```diff
@@ -143,8 +150,9 @@ class BackdropScaffold extends StatefulWidget {
   /// If null, the color is handled automatically according to the theme.
   final Color? frontLayerBackgroundColor;
 
-  /// Fraction of the available height the front layer will occupy,
-  /// when active (back layer concealed).  Clamped to (0, 1).
+  /// Fraction of available height the [frontLayer] will occupy when active (back layer concealed).
```

Comment:
can we make this a bit shorted? any idea? by few words - crossing recommend line width

Reply:
@WieFel

Reply:
For example:
Fraction of screen height the active (visible) [frontLayer] will occupy.

Reply:
let's keep it as it is, just break the line.

Reply:
done with - 710ead8